index.html

```diff
@@ -148,40 +148,16 @@ <h1 class="lets">My Recent Works</h1>
         </div><img src="img/circle-6.png" class="circle-6" alt="circle">
         </section>
 
-    <section class="contact-me">
+    <footer class="contact-me">
 <p class="lets">Let’s chat!</p>
 <p class="contact">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
-<form method="POST" class ="contactForm">
-    <div class="row">
-    <div class="small-fe">
-        <div>
-    <input type="text" class="form-control" name="firstname" id="firstname" placeholder="First Name">
-    </div>
-    </div>
-    <div class="small-fe">
-        <div>
-    <input type="email" class="form-control" name="lastname" id="lastname" placeholder="Last Name">
-    </div>
-    </div>
-
-    <div class="big-fe">
-    <div>
-    <input type="text" class="form-control" name="email" id="email" placeholder="Email">
-    </div>
-    </div>
-    <div class="big-fe">
-    <div>
-    <textarea name="message" class="form-massage" id="message" cols="30" rows="7" placeholder="Write Your Massage Here"></textarea>
-    </div>
-    </div>
-    <div class="big-fe">
-    <div>
+<form method="POST" class ="contactForm" action="https://formspree.io/f/mqkwgokn">
```

Comment:
[ OPTIONAL ] Please remove the horizontal scroll which makes the design a bit ugly and the form is not centered because of this problem. Kindly see the below screenshot.
![overflow](https://user-images.githubusercontent.com/35504149/130679198-062502e4-7513-434c-a812-445983ad0faf.png)